app/views/staff/organisations/show.html.haml

```diff
@@ -8,6 +8,18 @@
       %h1.govuk-heading-xl
         = @organisation_presenter.name
 
+  - if policy(:programme).create? && !@organisation_presenter.service_owner?
+    .govuk-grid-row
+      .govuk-grid-column-full
+        %h2.govuk-heading-m
+          = t("create_programme", scope: "page_content.organisation")
+
+        - @source_funds.each do |source|
+          = link_to t("form.button.activity.new_with_source_fund", name: source.name),
+            organisation_source_fund_children_path(@organisation_presenter, source.id),
+            method: :post, class: "govuk-button"
+          %br/
```

Comment:
I think these might look better aligned side by side with a `govuk-!-margin-right-4` class added, like this:

![image](https://user-images.githubusercontent.com/109774/107616155-6819da00-6c45-11eb-949f-a63658a08321.png)

Reply:
Yeah, I think you're right, was just following the mock up too closely 👍